pom.xml

```diff
@@ -930,6 +410,6 @@
         <version>${keycloak.version}</version>
         <scope>test</scope>
       </dependency>
-   </dependencies>
+    </dependencies>
```

Comment:
Totally minor obviously, but an extra space crept in here :)

Reply:
It's probably because we were missing a space before since it looks like indentation is correct in my IDE.

Reply:
:+1: